### PR TITLE
Constant-time encoding

### PR DESCRIPTION
**Pull request by paragonie-scott**
See #761

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "require-dev": {
         "phing/phing": "~2.7",
         "phpunit/phpunit": "~4.0",
+        "paragonie/constant_time_encoding": "^0.5",
```
Comment (GrahamCampbell) [inline composer.json]: :-1: until this is stable

**Comment by GrahamCampbell** [comment]
Surely this would be better targeted at 2.x, rather than 3.x, since it could be a year before 3.0 gets released.

**Comment by paragonie-scott** [comment]
Okay, I'll send a superseding PR against 2.x once I've gotten enough independent review to feel confident about tagging 1.0.0.

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by terrafrost** [comment]
One thing we need to keep in mind with doing Composer stuff for the 2.0 branch is that 2.0 doesn't necessarily require composer. What it requires is a PSR-4 autoloader whereas composer does PSR-0, classmap and files autoloading as well. I guess that doesn't apply so much to paragonie/constant_time_encoding as it does to paragonie/random_compat but it is, none-the-less, worth keeping in mind.

With 3.0, in contrast, we're free to do mostly whatever we want as there's no BC we need to worry about.

**Comment by paragonie-scott** [comment]
By the way, we tagged v1.0 and v2.0: https://github.com/paragonie/constant_time_encoding/releases

**Pull request closed without merging**